scout/load/hpo.py

```diff
@@ -118,6 +118,13 @@ def load_hpo_terms(
 
         hpo_term["genes"].add(hgnc_id)
 
+    if not hpo_terms:
+        LOG.error("No HPO terms fogund. Aborting update without dropping HPO term collection.")
```

Comment:
```suggestion
        LOG.error("No HPO terms found. Aborting update without dropping HPO term collection.")
```

Reply:
Ahem, just one more of those editor focus things I'm sure. 😁

Reply:
No worries, that's what the reviews are for! 😄